BlockSettleUILib/Trading/RFQDialog.cpp

```diff
@@ -156,7 +156,10 @@ void RFQDialog::logError(bs::error::ErrorCode code, const QString &errorMessage)
    logger_->error("[RFQDialog::logError] {}", errorMessage.toStdString());
 
    if (bs::error::ErrorCode::TxCancelled != code) {
-      MessageBoxBroadcastError(errorMessage, code, this).exec();
+      // Do not use this as the parent as it will be destroyed when RFQDialog is closed
+      QMetaObject::invokeMethod(qApp, [code, errorMessage] {
```

Comment:
Why do you use qApp's context instead of `this`? RFQDialog is a widget and is unlikely was created in other thread than main.

Reply:
RFQDialog is destroyed soon after the error (not sure when exactly). And when QObject is destroyed all messages to the object removed from queue.